src/copick_utils/writers/write.py

```diff
@@ -0,0 +1,173 @@
+from typing import Any, Dict, List
+import ome_zarr.writer
+import zarr
+
+def tomogram(
+    run,
+    inputVol,
+    voxelSize=10,
+    tomo_algorithm="wbp"
+    ):
+    """
+    Write a OME-Zarr segmentation into a Copick Directory.
+
+    Parameters:
+    - run: The run object, which provides a method to create a new segmentation.
+    - segmentation: The segmentation data to be written.
+    - voxelsize (float): The size of the voxels. Default is 10.
+    """
+
+    # Create a new segmentation or Read Previous Segmentation
+    vs = run.get_voxel_spacing(voxelSize)
+    if vs is None:
+        vs = run.new_voxel_spacing(
+            voxel_size=voxelSize,
+        )
+        tomogram = vs.new_tomogram(tomo_algorithm)
+    else:
+        tomogram = vs.get_tomogram(tomo_algorithm)
+        
+    # Write the zarr file
+    loc = tomogram.zarr()
+    root_group = zarr.group(loc, overwrite=True)
+
+    ome_zarr.writer.write_multiscale(
```

Comment:
I had added this method for convenience, maybe it makes sense to use it here instead of a separate implementation:

https://github.com/copick/copick/blob/90ec8a25e201a7439e70ef24d1642e244bc01e76/src/copick/models.py#L1344